gradle-plugin/src/main/kotlin/xyz/block/artifactswap/ArtifactSwapModuleSelectionValueSource.kt

```diff
@@ -54,7 +55,8 @@ internal abstract class ArtifactSwapModuleSelectionValueSource :
     val config = parameters.config.get()
 
     // Read project lists
-    val ideProjectsList = SpotlightProjectList.ideProjects(rootDir).read()
+    val allProjectsList = SpotlightProjectList.allProjects(rootDir).read()
+    val ideProjectsList = SpotlightProjectList.ideProjects(rootDir) { allProjectsList }.read()
```

Comment:
this fixed the issue we saw with having wildcard paths in ide-projects.txt. they need the list of all projects to be expanded properly